index.html

```diff
@@ -39,6 +39,88 @@ <h2>I'm a Software Developer</h2>
         </ul>
       </nav>
     </section>
+
+    <section class="recent-works-section">
+      <div id="my-recent-works">
+        <h2>My Recent Works</h2>
+      </div>
+      <div class="grid-container">
+
+        <div class="grid-item1">
+          <div class="featured-work">
+            <img src="resources/images/Placeholder1.svg" />
+            <div class="title-featured">Multi-post Stories</div>
+            <div class="featured-content">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's 
+              standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+            </div>
+            <div class="featured-tags">
+              <ul>
+                <li>CSS</li>
+                <li>html</li>
+                <li>Javascript</li>
+              </ul>
+            </div>
+            <button class="featured-btn">See Project</button>
+          </div>
+        </div>
+
+        <div class="work-post">
+          <div class="title-work">Profesional Art Printing Data</div>
+          <div class="work-content">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</div>
+          <div class="work-tags">
+            <ul>
+              <li>CSS</li>
+              <li>html</li>
+              <li>Javascript</li>
+            </ul>
+          </div>
+          <button class="work-btn">See Project</button>
+        </div>
+      </div>
+    </section>
+
+    <section class="about-me">
+      <div class="about-me-text">About me</div>
```

Comment:
- [ ] Please make sure that your `About me` text is wrapped by an `h1` instead of a `div`, it's because of semantic reasons to use `headings` for your titles. 👍️